lib/ruby_indexer/lib/ruby_indexer/collector.rb

```diff
@@ -350,5 +352,24 @@ def handle_attribute(node, reader:, writer:)
         @index << Entry::Accessor.new("#{name}=", @file_path, loc, comments, @current_owner) if writer
       end
     end
+
+    sig { params(node: Prism::CallNode).void }
+    def handle_include(node)
+      return unless @current_owner
+
+      arguments = node.arguments&.arguments
+      return unless arguments
+
+      names = arguments.filter_map do |node|
+        if node.is_a?(Prism::ConstantReadNode) || node.is_a?(Prism::ConstantPathNode)
+          T.let(node.full_name, String)
```

Comment:
Do you need this `T.let` here?

Reply:
Oh, it shouldn't be an issue that it is untyped. Don't worry about it. Can you please remove the `T.let` and just leave `node.full_name`?